.github/workflows/docs.yaml

```diff
@@ -0,0 +1,133 @@
+name: Docs
+
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'docs/**'
+    - '.github/workflows/docs.yaml'
```

Comment:
We also need to trigger it on pushes to release branches

Reply:
yeah, you are right, although the design of checking random tip of the other branches is not great when all of them publish (possible races), it is what we have.